docs/commands/blueprints/render.md

```diff
@@ -98,62 +167,193 @@ The resources are written as files in the following directory structure to the g
     └── state
 ```
 
-### Using Data from the Component Descriptor
+### Rendering with Export Templates
```

Comment:
I miss some description about what export templates are and what you could achieve with them.